src/player.h

```diff
@@ -1139,6 +1139,21 @@ class Player final : public Creature, public Cylinder
 		void forgetInstantSpell(const std::string& spellName);
 		bool hasLearnedInstantSpell(const std::string& spellName) const;
 
+		bool startLiveCast(const std::string& password) {
+			return client != nullptr && client->startLiveCast(password);
+		}
+
+		bool stopLiveCast() {
+			return client != nullptr && client->stopLiveCast();
+		}
+
+		bool isLiveCaster() const {
+			return client != nullptr && client->isLiveCaster();
+		}
+
```

Comment:
`client != nullptr` could be shortened to just `client`, no? Most of the code does it anyway.